src/features/view-subdomains/SubdomainTable/SubdomainTable.tsx

```diff
@@ -39,16 +48,28 @@ export const SubdomainTable: FC<SubdomainTableProps> = ({
 
 	return (
 		<>
-			<div
-				style={{
-					display: 'flex',
-					justifyContent: 'flex-end',
-					color: 'black',
-				}}
-			>
-				<button onClick={() => changeView(false)}>Row</button>
-				<button onClick={() => changeView(true)}>Grid</button>
+			{/* todo: leave or add controls to table component */}
+			<div className={styles.Controls}>
+				<div className={styles.SearchBarContainer}>
+					<SearchBar
```

Comment:
Search functionality to be wired up in a different PR - shell component for now

Reply:
Nice! Don't worry about search for now - it will be built into the table component.